lib/Distribution/Sle/AgamaDevel.pm

```diff
@@ -17,12 +17,17 @@ use Yam::Agama::Pom::GrubEntryEditionPage;
 use Yam::Agama::Pom::Sle::AgamaUpAndRunningPage;
 use Yam::Agama::Pom::RebootPage;
 
+use Utils::Architectures;
+
 sub get_grub_menu {
     return Yam::Agama::Pom::GrubMenuPage->new();
 }
 
 sub get_grub_entry_edition {
-    return Yam::Agama::Pom::GrubEntryEditionPage->new();
+    return is_ppc64le() ? Yam::Agama::Pom::GrubEntryEditionPage->new({
+            number_kernel_line => 3,
+            max_interval => utils::VERY_SLOW_TYPING_SPEED})
+      : Yam::Agama::Pom::GrubEntryEditionPage->new();
```

Comment:
This better in another PR, so this one would be only about s390x, in case we need review from others outside the squad would be more clear.

Reply:
1825d85eea6b903a2c986748234f860179c5f973